users/miketronic/rules.mk

```
@@ -0,0 +1 @@
+SRC += miketronic.c tapdances.c process_records.c
```

If you want the tap dance stuff to be conditionally added, you may want to do this: 
```suggestion
SRC += miketronic.c process_records.c

ifeq ($(strip $(TAP_DANCE_ENABLE)), yes)
    SRC += $(USER_PATH)/tapdances.c
endif
```

This way, it's only compiled if the feature is enabled. 

(also, in general, it may be a good idea to use the full-ish paths like this, and it may help reduce collisions and other weird issues with compilation)